python/testapi/core/render/test_exporter.py

```diff
@@ -68,5 +86,10 @@ def _check_frames(self) -> None:
                 validator.validate_file(test, ref)
             except RuntimeError as e:
                 errors.append(str(e))
-        self.output.rmdir()
+        self._cleanup()
         self.assertFalse(errors)
+
+    def _cleanup(self) -> None:
```

Comment:
I think it would be safer to use ${TMPDIR} if setted (or /tmp if the former is not set) than manually deleting files and folders.